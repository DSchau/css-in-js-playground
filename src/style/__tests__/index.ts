import * as STYLE from '../';

test('it contains expected exports', () => {
  expect(Object.keys(STYLE)).toMatchSnapshot();
});
