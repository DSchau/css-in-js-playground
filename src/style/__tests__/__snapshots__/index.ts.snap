// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`it contains expected exports 1`] = `
Array [
  "SLIDE_UP",
  "SANS_SERIF",
  "SERIF",
  "THEME",
]
`;
